backend/lib/models/Comment.js

```diff
@@ -50,5 +50,23 @@ CommentSchema.index({
 
 const Comment = model("Comment", CommentSchema);
 
-exports.model = Comment;
-exports.schema = CommentSchema;
+function updateAuthorName(authorID, newAuthorName) {
+  return Comment.where(
+    { "author.authorId": authorID },
```

Comment:
Author doesn't have `author` prefix anymore. 
Should be `author.id` and `author.name` at the updates here.

Reply:
Probably should be careful to not conflict with #378 too